db/schema.rb

```diff
@@ -966,12 +953,8 @@
   add_foreign_key "posts", "users", column: "target_user_id"
   add_foreign_key "profile_links", "profile_link_sites"
   add_foreign_key "profile_links", "users"
-  add_foreign_key "quote_likes", "quotes"
-  add_foreign_key "quote_likes", "users"
-  add_foreign_key "quotes", "characters"
   add_foreign_key "reports", "users"
   add_foreign_key "reports", "users", column: "moderator_id"
-  add_foreign_key "review_likes", "reviews"
```

Comment:
I'm not sure where all these came from, tbh, but I keep getting the same so I'll allow it